scanhub-ui/src/components/PatientTable.tsx

```diff
@@ -5,82 +5,49 @@
+
   const navigate = useNavigate()
 
+  const columns: GridColDef<PatientOut>[] = [
+    { field: 'id', headerName: 'ID', width: 200, editable: false },
+    { field: 'name', headerName: 'Name', width: 200, editable: false },
+    {
+      field: 'sex',
+      type: 'singleSelect',
+      headerName: 'Sex',
+      width: 200,
+      editable: false,
+      valueOptions: ['m', 'w', 'o'],
+    },
+    { field: 'birth_date', headerName: 'Birthday', width: 200, editable: false },
+    { field: 'status', headerName: 'Status', width: 200, editable: false },
+  ]
+
   return (
     <Sheet variant='outlined' sx={{ p: 1, borderRadius: 'sm' }}>
-      <Table hoverRow borderAxis='xBetween' color='neutral' size='sm' stickyHeader variant='plain'>
-        <thead>
-          <tr>
-            <th style={{ width: '4%' }}>ID</th>
-            <th>Name</th>
-            <th style={{ width: '4%' }}>Sex</th>
-            <th style={{ width: '8%' }}>Birthday</th>
-            <th>Issuer</th>
-            <th>Status</th>
-            <th>Comment</th>
-            <th style={{ width: '8%' }}>Admission</th>
-            <th style={{ width: '8%' }}>Updated</th>
-          </tr>
-        </thead>
-
-        <tbody>
-          {props.patients.map((patient) => (
-            <tr
-              key={patient.id}
-              onClick={() => {
-                navigate(`/${patient.id}`)
-              }}
-            >
-              <td>{patient.id}</td>
-              <td>{patient.name}</td>
-              <td>{patient.sex}</td>
-              <td>{patient.birth_date}</td>
-              <td>{patient.issuer}</td>
-              <td>{patient.status}</td>
-              <td>{patient.comment}</td>
-              <td>{new Date(patient.datetime_created).toDateString()}</td>
-              <td>{patient.datetime_updated ? new Date(patient.datetime_updated).toDateString() : '-'}</td>
-
-              {/* <td>
-                <Box sx={{ display: 'flex', gap: 1 }}>
-                  <Button
-                    size='sm'
-                    sx={{ zIndex: 'snackbar' }}
-                    variant='soft'
-                    endDecorator={<KeyboardArrowRight />}
-                    color='neutral'
-                    onClick={() => {
-                      navigate(`/patients/dcmview/${patient.id}`)
-                    }}
-                  >
-                    View
-                  </Button>
-                  <Button
-                    size='sm'
-                    sx={{ zIndex: 'snackbar' }}
-                    variant='soft'
-                    endDecorator={<KeyboardArrowRight />}
-                    color='neutral'
-                    onClick={() => {
-                      navigate(`/patients/${patient.id}`)
-                    }}
-                  >
-                    Acquire
-                  </Button>
-                </Box>
-              </td> */}
-            </tr>
-          ))}
-        </tbody>
-      </Table>
+      <DataGrid
```

Comment:
DataGrid not scaled to full width

Reply:
Yes, because that space is not needed yet.

Reply:
I would still scale the grid to full width with some margin around as before. Also the user data grid on my notebook had different spacing on the left and the right.

Reply:
Ok